sample/lib/widgets/showcase/sample_message_bubble.showcase.dart

```diff
@@ -53,6 +53,11 @@ class SampleMessageBubbleShowcase extends StatelessWidget {
     return Obx(
       () => Column(
         children: [
+          DSEndCallsMessageBubble(
```

Comment:
Add a right aligned bubble as well and it would be so much helpful if you could setup real examples with audio, loading and error states.